File: task-03/Haskel/prime_number.hs
import Control.Monad

isPrime :: Integer -> Bool
isPrime n
    | n <= 1    = False
    | n <= 3    = True
    | otherwise = all (\x -> n `mod` x /= 0) [2..isqrt n]
    where isqrt = floor . sqrt . fromIntegral

main :: IO ()
main = do
    putStrLn "Enter the  limit:"
    input <- getLine
    let n = read input :: Integer
    putStrLn $ "Prime numbers are" ++ show n ++ ":"
    forM_ [2..n] $ \i -> when (isPrime i) (print i)

File: task-03/Python/prime_number.py
import  math
n = int(input("Enter a num: \n"))
if(n < 2):
    print("No prime")
else:
    for i in range (2,n+1):
        flag = 0
        for j in range( 2,int(math.sqrt(i))+1):
            if(i % j == 0):
                flag = 1
                break
        
        if(flag == 0 ):
            print(i, end= " ")
